src/main/cli/process.ts

```diff
@@ -0,0 +1,70 @@
+// ==LICENSE-BEGIN==
+// Copyright 2017 European Digital Reading Lab. All rights reserved.
+// Licensed to the Readium Foundation under one or more contributor license agreements.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file exposed on Github (readium) in the project repository.
+// ==LICENSE-END==
+
+import * as debug_ from "debug";
+import { app } from "electron";
+import { ICli, ICliParam } from "readium-desktop/main/cli/commandLine";
+import { Arguments } from "yargs";
+
+declare const __APP_NAME__: string;
+declare const __APP_VERSION__: string;
+
+// Logger
+const debug = debug_("readium-desktop:cli");
+
+export function printHelp(commandLine: ICli[], argv: Arguments) {
+
+    if (argv.help || argv.h) {
+        process.stdout.write(`${__APP_NAME__} ${__APP_VERSION__}\n`);
+        process.stdout.write("\n");
+        process.stdout.write(`Usage: ${argv.$0} [options][paths...]\n`);
+        process.stdout.write("\n");
+        process.stdout.write(`Options\n`);
+        commandLine.map((v) => {
+            if (v.help.length) {
+                process.stdout.write(` ${v.help[0]}\t\t${v.help[1] ? v.help[1] : ""}\n`);
+            }
+        });
+        app.exit(0);
+        process.exit(0);
```

Comment:
`process.exit(0);` is not necessary after `app.exit(0);`

Reply:
done 64c63d2